service/worker/migration/force_replication_workflow.go

```diff
@@ -26,15 +26,31 @@ package migration
 
 import (
 	"errors"
+	"fmt"
 	"time"
 
 	commonpb "go.temporal.io/api/common/v1"
+	enumspb "go.temporal.io/api/enums/v1"
 	"go.temporal.io/api/workflowservice/v1"
 	"go.temporal.io/sdk/temporal"
 	"go.temporal.io/sdk/workflow"
 )
 
 type (
+	TaskQueueUserDataReplicationParams struct {
+		// PageSize for the SeedReplicationQueueWithUserDataEntries activity
+		PageSize int
+		// RPS for enqueuing of replication tasks
```

Comment:
worth noting the units: one "request" here is one page of tq user data

Reply:
True, I can fix the comment in a followup PR.